src/index.ts

```diff
@@ -1,9 +1,43 @@
+import { RegisterRoutes } from "./generated/routes";
 import { EXPRESS_PORT, NODE_ENV } from "@config";
+// TODO: enable when disableWriteWhenNotDev is available
+// import { disableWriteWhenNotDev } from "@disableWriteWhenNotDev";
 import { runMigrations } from "@db/migrations";
-import { createExpressServer } from "@createExpressServer";
+import express, { NextFunction } from "express";
+import openapi from "./openapi";
+import { pinoHttp } from "pino-http";
+import { jwtErrorHandler } from "@auth/jwt-error";
+import rateLimit from "express-rate-limit";
 
 async function startServer() {
-  const app = createExpressServer();
+  const app = express();
```

Comment:
Looks like the rebase went wrong here.
Cod below should be moved back to `createExpressServer`

Reply:
I'll do that.